src/app/ui/entries_list/mod.rs

```diff
@@ -238,12 +239,12 @@ impl<'a> EntriesList {
         let place_holder = Paragraph::new(place_holder_text)
             .wrap(Wrap { trim: false })
             .alignment(Alignment::Center)
-            .block(self.get_list_block(has_filter));
+            .block(self.get_list_block(has_filter, entries_len));
 
         frame.render_widget(place_holder, area);
     }
 
-    fn get_list_block(&self, has_filter: bool) -> Block<'a> {
+    fn get_list_block(&self, has_filter: bool, entries_len: usize) -> Block<'a> {
```

Comment:
As mentioned before I would make this optional `entries_len: Option<usize>` and make the code snippets on line 271 something like:
```rust
        match (entries_len, self.state.selected().map(|v| v + 1)) {
            (Some(entries_len), Some(selected)) => {
                block.title_bottom(Line::from(format!("{selected}/{entries_len}")).right_aligned())
            }
            _ => block,
        }
```